models/staging/hubspot_staging/stg_rag_hubspot.yml

```diff
@@ -96,6 +96,39 @@ models:
       - name: source_relation
         description: The source of the record if the unioning functionality is being used. If it is not this field will be empty.
 
+  - name: stg_rag_hubspot__engagement
+    description: Each record represents an engagement 
+    columns:
+      - name: engagement_id
+        description: The ID of the engagement.
+        tests:
+          - not_null
+          - unique
+      - name: engagement_type
+        description: One of NOTE, EMAIL, TASK, MEETING, or CALL, the type of the engagement.
+      - name: is_active
+        description: >
+          Whether the engagement is currently being shown in the UI.
+
+          PLEASE NOTE - This field will only be populated for pre HubSpot v3 API versions. This field is only included to allow for backwards compatibility between HubSpot API versions. This field will be deprecated in the near future.
+      - name: occurred_timestamp
+        description: >
+          A timestamp in representing the time that the engagement should appear in the timeline.
+
+          PLEASE NOTE - This field will only be populated for pre HubSpot v3 API versions. This field is only included to allow for backwards compatibility between HubSpot API versions. This field will be deprecated in the near future.
+      - name: created_timestamp
+        description: >
+          This field marks the call's time of creation and determines where the call sits on the record timeline. You can use either a Unix timestamp in milliseconds or UTC format. 
+
+          PLEASE NOTE: This field will only be populated for pre HubSpot v3 API versions. This field is only included to allow for backwards compatibility between HubSpot API versions. This field will be deprecated in the near future.
+      - name: owner_id
+        description: >
+          The ID of the engagement's owner.
+
+          PLEASE NOTE - This field will only be populated for pre HubSpot v3 API versions. This field is only included to allow for backwards compatibility between HubSpot API versions. This field will be deprecated in the near future.
+      - name: portal_id
```

Comment:
Don't forget to document `source_relation`!

Reply:
thanks! updated